### DISCUSSION TITLE
After opening right-click menu on the taskbar the icons there don't launch from the first click

### DISCUSSION THREAD

**Discussion by SquidBow** [valinet/ExplorerPatcher#3646] [Q&A]
When I open the right-click menu on the taskbar and than try to launch smth from the taskbar without closing the menu it just doesn't. The problem is ecpesialy evidant with the start button, whitch becomes smaller and stays like that nomater what I do till I open the start menu by clicking it (windows key doesn't fix the problem eather)

![Знімок екрана 2024-09-06 204517](https://github.com/user-attachments/assets/fbb8ba6e-573c-49ff-bf9a-956e36eabc17)
![Bug1](https://github.com/user-attachments/assets/0452c067-2ae4-4555-a8ee-d6dfded7df7c)

**Comment by pyrates999** [comment]
So this is happening?

1. right click on taskbar
2. click on any of the icons on the taskbar like the start menu
3. nothing happens

Is that correct?

**Reply by SquidBow** [reply]
Normal
![photo_2_2024-09-07_08-01-26](https://github.com/user-attachments/assets/796184f4-0a09-4175-8269-dc5092e93e20)


Bugged
![photo_1_2024-09-07_08-01-26](https://github.com/user-attachments/assets/ca0b2320-a8e7-4eff-9469-89ecddbbdc64)

**Comment by SquidBow** [comment]
Besides the visual bug nothing
And btw, i just notised that prntscr didn't capture this visual bug, lol. I will take a photo with my phone than, but only in a few hours, don't have access to my laptop rn

**Comment by Amrsatrio** [comment]
Your start menu crashed.

**Reply by SquidBow** [reply]
Oh, so thats what is happening. But the problem is a bit different, like if I click any icon in this sonario it doesn't open, just closes the menu

**Comment by pingognosis** [comment]
I have a similar problem: right-clicking on (e.g.) the Firefox taskbar icon used to launch a menu; now it does nothing. Right-clicking on the desktop launches the Windows context menu normally (i.e., it's not a problem with the touchpad).
